src/cluster.rs

```diff
@@ -171,34 +171,25 @@ impl ClusterConnection {
     }
 
     /// Check that all connections it has are available (`PING` internally).
-    pub fn check_connection(&mut self) -> bool {
+    pub fn check_connection(&mut self) -> RedisResult<()> {
```

Comment:
This changes the public API so I would have to make a breaking release which seems overkill. Could you make the `Result` returning method private and use that internally and keep this public version with `bool`? Then I can make a semver compatible release

Reply:
This is not possible because I would also have to revert the Connection trait changes as that is public.
That would be a major problem because everything goes through that trait and I cant add a private method to a trait.